src/components/AccountForm/accountForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+import BaseForm from '../BaseForm/baseForm';
+import PropTypes from 'prop-types';
+
+const AccountForm = ({ accountId = null }) => {
+    const [formAccount, setFormAccount] = useState({
+        name: '',
+        amount: 0,
+        type: 1
+    });
+
+    const updateFormAccount = (newFormAccount) => {
+        setFormAccount(newFormAccount);
+    }
+
+    const isEdit = accountId !== null;
+
+    const sendRequestToUpdate = () => {
+        alert('update account');
+    }
+
+    const onSubmit = (data) => {
+        if (isEdit)
+            sendRequestToUpdate();
+        setFormAccount(data);
+    };
+
+    return (
+        <div className="App">
+            <h1>{isEdit ? 'Edit Account' : 'Add Account'}</h1>
+            <BaseForm initialValues={{
+                    name: '',
+                    amount: 0,
+                    type: 1
+                }}
+                onSubmit={onSubmit}
+                submitLabel={ isEdit ? 'Actualizar cuenta' : 'Agregar cuenta'}
+                formInputs={[
+                    {
```

Comment:
With the simplified solution you just need to add a new prop to pass the `interface` name, in this case `input` or `select`